validator/node/node.go

```diff
@@ -209,23 +212,6 @@ func selectKeyManager(ctx *cli.Context) (keymanager.KeyManager, error) {
 		opts = string(fileopts)
 	}
 
-	if manager == "" {
```

Comment:
Can we actually keep this in? This would be helpful

Reply:
The deprecated vars have gone away so there's nothing for this code to do.

Would you like me to set a default for the keymanager?